integration-tests/project/Dependencies.scala

```diff
@@ -0,0 +1,27 @@
+import sbt._
+
+object Dependencies {
+
+  object Version {
+    val cats = "1.5.0"
```

Comment:
1.6.0 is out
```suggestion
    val cats = "1.6.0"
```

Reply:
Have to see if this is compatible with http4s